### DISCUSSION TITLE
When will winget be part of the non-insider windows experience?

### DISCUSSION THREAD

**Discussion by vinczemarton** [microsoft/winget-cli#1301] [Q&A] [answered]
I'm not the biggest fan of beta software. When winget was announced I kept an eye on it and waited for it to become part of the stable windows ecosystem.

Since the software got to 1.0 I thought that this will be the time when it starts to be available outside the insiders program, but it is still just for insiders.

Plans for the stable rollout are unclear for me as I cannot find any statement regarding when will it happen or will it ever happen.

**Comment by jedieaston** [comment, marked as answer]
This has been covered in other places, but the current plan is September.

Related discussion message: https://github.com/microsoft/winget-cli/discussions/988#discussioncomment-793667

It is to be considered stable at this time, but they don't want to push it to enterprises which aren't expecting it.

**Reply by DarthXibalba** [reply]
Is WinGet going to be apart of the Windows 11 21H1? This might not be the best place to post this, but I work for one of MS's largest partners and I'm pushing to distribute winget as apart of our software stack.
Not sure if I should I could get in touch with someone from your team regarding redistribution rights or if I should leave my email in this comment.

**Reply by jedieaston** [reply]
I'm sure @denelon can get you more info but as of Windows 11 21H2 it is included in the default install. If you're targeting Windows 11 you shouldn't need to distribute it. Windows 10 is going to get it in September I believe.

**Reply by denelon** [reply]
All Windows insider builds have a version of the App Installer with the Windows Package Manager. We are targeting the end of September for the release to Windows 10 (1809+) builds.
